python/chronos/test/bigdl/chronos/data/experimental/test_xshardstsdataset.py

```diff
@@ -192,6 +192,33 @@ def test_xshardstsdataset_roll_multiple_id(self):
         x = np.concatenate([collected_numpy[i]['x'] for i in range(len(collected_numpy))], axis=0)
         assert x.shape == ((50-lookback-horizon+1)*2, lookback, 2)
 
+    def test_xshardstsdataset_impute_multiple_id(self):
+        shards_multiple = read_csv(os.path.join(self.resource_path, "multiple.csv"))
```

Comment:
maybe generate a df that needs to be imputed(have missing values). dump it to tmp dir and load it back through `read_csv`. I think "multiple.csv" does not have any missing values.